SystemInformer/runas.c

```diff
@@ -2399,9 +2399,9 @@ NTSTATUS PhpRunFileProgram(
 
             if (PhGetOwnTokenAttributes().Elevated)
             {
-                // Note: This is needed to workaround a severe bug with PROC_THREAD_ATTRIBUTE_PARENT_PROCESS 
-                // where the process and token security descriptors are created without an ACE for the current user, 
-                // owned by the wrong user and with a High-IL when the process token is Medium-IL 
+                // Note: This is needed to workaround a severe bug with PROC_THREAD_ATTRIBUTE_PARENT_PROCESS
+                // where the process and token security descriptors are created without an ACE for the current user,
+                // owned by the wrong user and with a High-IL when the process token is Medium-IL
```

Comment:
Don't change these lines

Reply:
But this is whitespacess!

Reply:
You'll be getting blamed for these comments/lines by changing them. For example:

https://github.com/winsiderss/systeminformer/blame/44d9c6b138236b9af9812c59ccf309189fb2ad81/SystemInformer/runas.c#L2402

https://github.com/winsiderss/systeminformer/blame/44d9c6b138236b9af9812c59ccf309189fb2ad81/SystemInformer/runas.c#L1936

![image](https://user-images.githubusercontent.com/1306177/188504969-ad24bf79-5994-4631-9365-d14e551e6627.png)

All these lines (and the others above) are somewhat security related. You don't want to get blamed for these in 5-10 years down the road if they're incorrect 😉